src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -0,0 +1,312 @@
+    "dxDepartment": "product",
+    "dxProduct": "Hello"
+  },
+
+  "resources": [
+    {
+      "id": "host1",
+      "display_name": "Hello-Host1",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3.micro"
+    }
+  ],
+
+  "instrumentations": {
+    "resources": [
+      {
+        "id": "nr_infra",
+        "resource_ids": ["host1"],
+        "provider": "newrelic",
+        "source_repository": "-b main git@github.com:newrelic/demo-newrelic-instrumentation.git",
+        "deploy_script_path": "deploy/linux/roles",
+        "version": "1.12.1"
+      }
+    ],
+    "services": [
+      {
+        "id": "nr_node_agent",
+        "service_ids": ["app1"],
+        "provider": "newrelic",
+        "source_repository": "-b main git@github.com:newrelic/demo-newrelic-instrumentation.git",
+        "deploy_script_path": "deploy/node/linux/roles",
+        "version": "6.11.0"
+      }
+    ]
+  }
+}
+```
+
+</Step>
+
+<Step>
+
+## Your newly deployed stack!
+When completed successfully, you should see output similar to what's below. Of course, your URLs will be different, but go ahead and open up a browser on the 'app1' url.
+
+```bash
+[INFO] Executing Deployment
+[✔] Parsing and validating Deployment configuration success
+[✔] Provisioner success
+[✔] Installing On-Host instrumentation success
+[✔] Installing Services and instrumentations success
+[INFO] Deployment successful!
+
+Deployed Resources:
+
+  host1 (aws/ec2):
+    ip: 52.90.86.109
+    services: ["app1"]
+    instrumentation:
+       nr_infra: newrelic v1.12.1
+
+
+Installed Services:
+
+  app1:
+    url: http://52.90.86.109:5001
+    instrumentation:
+       nr_node_agent: newrelic v6.11.0
+
+Completed at 2020-08-20 15:11:14 +0000
+```
+
+</Step>
+
+<Step>
+
+## Generate Traffic
+The simple node application that was deployed contains a very simple web page that looks similar to the screen shot below. Follow the instructions on this page to generate traffic that will be sent to New Relic.
+![Hello World Web Site](../../images/automate-helloworld/helloworld-automationdemo.png)
+
+</Step>
+
+<Step>
+
+## Check out these tags!
+Let's head on over to New Relic. We'll use the [search redirect](https://one.newrelic.com/redirect/search/Hello) to quickly find the resources we deployed. (Folks in our European data center should use this [link](https://one.eu.newrelic.com/redirect/search/Hello).)
+
+![Search Redirect](../../images/automate-helloworld/global-search.png)
+
+</Step>
+
+<Step>
+Once you are in New Relic, you can select either the host or the app that was deployed. By clicking on the 'i' icon next to the name, you'll open the metadata and tags modal. Notice that all the tags in the "global_tags" section are applied to both entities. 
+
+![Search Redirect](../../images/automate-helloworld/host-tags.png)
+
+![Search Redirect](../../images/automate-helloworld/app-tags.png)
+
+</Step>
+
+<Step> 
+## Extra Credit: Create your own configuration file
```

Comment:
```suggestion
<Step> 

## Extra Credit: Create your own configuration file
```

It looks like `## Extra Credit` is output to the screen rather than an `h2`. The extra space here should fix that.